javascripts/govuk/show-hide-content.js

```diff
@@ -80,22 +79,23 @@
     // Handle radio show/hide
     function handleRadioContent ($control, $content) {
 
-      // All radios in this group
-      var selector = selectors.radio + '[name=' + escapeElementName($control.attr('name')) + ']'
+      // All radios in this group which control content
+      var selector = selectors.radio + '[name=' + escapeElementName($control.attr('name')) + '][aria-controls]'
```

Comment:
Thanks @tombye, that's great.

Forgot I wrote a test to cover that very scenario!
https://github.com/alphagov/govuk_frontend_toolkit/pull/315/files#diff-1cb888219674ec4080d3a7d2bdca5117R183

Still works as intended then 👍